workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs

```diff
@@ -31,7 +31,7 @@ public EntityGameObjectCreator(World world, Dictionary<string, GameObject> initi
                 cachedPrefabs[prefabPath] = prefab;
             }
 
-            var gameObject = GameObject.Instantiate(prefab, position, rotation);
+            var gameObject = Object.Instantiate(prefab, position, rotation);
```

Comment:
What's the reason for this change?

Reply:
Resharper complains about accessing a super-class' method via a subclass. I can also just suppress the warning

Reply:
Hmm - I'd say `GameObject.Instantiate...` is the more canonical way in Unity-land. But if Resharper complains then its probably fine